addons/bus/websocket.py

```diff
@@ -613,7 +618,7 @@ def _trigger_lifecycle_event(self, event_type):
         """
         if not self.__event_callbacks[event_type]:
             return
-        with closing(acquire_cursor(self._db)) as cr:
+        with acquire_cursor(self._db) as cr:
```

Comment:
Closing is gone, it used to rollback the cursor, now it commits it. Why dit it change?

Reply:
Isn't closing redundant with the context manager? It already was but since I worked on `acquire_cursor` I cleaned it up.

Reply:
If you call `retrying` at some point in the process, then you basically "delegates" the cursor management to that function. Hence you can just close the cursor here has everything has been commit already. But it is not *wrong* to commit it as well here. It wouldn't has bothered me in master, here we are in stable :/